k-distribution/pom.xml

```
@@ -122,6 +122,7 @@
               <target>
                 <exec dir="${ktestDir}" executable="make" failonerror="true">
                   <arg value="-j12" />
+                  <arg value="--output-sync" />
```

```suggestion
                  <arg value="--output-sync=recurse" />
```

https://www.gnu.org/software/make/manual/html_node/Parallel-Output.html

---

You'll want to change the Jenkinsfile where in calls `make` in the tutorial with teh same update.